reference/getting-started/cc/introduction-for-creative-coders.md

```diff
@@ -1,37 +1,37 @@
 # Introduction for Creative Coders
 
-What you can do with the various existing creative coding frameworks, is often roughly the same. What's mostly different is the way in which things are done. vvvv has a history spanning since the early twothousands. With the newer version "vvvv gamma" you get to use the results of many years of research into creating a convenient and fast to use visual live-programming environment.
+What you can do with various existing creative coding frameworks, is often roughly the same. What's mostly different is the way in which things are done. vvvv has a history spanning since the early twothousands. With the newer version "vvvv gamma" you get to use the results of many years of research into creating a convenient and fast to use visual live-programming environment.
 
 ## What people are using vvvv for
 vvvv is used in a wide range of projects covering topics like: generative-design, interaction-design, data-visualization, computer-vision, VR, show-control, physical-computing, machine-learning and generally for any kind of quick prototyping.
 Have a look at the [Gallery](https://visualprogramming.net/#Gallery) to see some specific examples.
 
 ---
 
-Depending on where you come from, vvvv offers different benefits:
+Depending on whether you already have experience with [text based coding](#coming-from-text-based-coding) or with [other visual programming environments](#coming-from-other-visual-programming-environments), vvvv offers different benefits:
 
 ## Coming from text based coding
 If you're familiar with text based creative coding, like:
 * [Cinder](https://www.libcinder.org) or [OpenFrameworks](https://openframeworks.cc)
-* [Nannou](https://nannou.cc) or [rin](https://rin.rs)  
+* [Nannou](https://nannou.cc) or [rin](https://rin.rs)
 * [Processing](https://processing.org) or [p5.js](https://p5js.org)
 * [OpenRNDR](https://openrndr.org)
 
 ... then the following could be interesting for you:
-  
-### Save time
 
-vvvv does not have the classic time-consuming edit-compile-run cycle. For every change you make to your vvvv program, you'll get instant results! We call this **live-programming** and are using a state hot-reload approach: Every change is instantly compiled under the hood without you having to trigger it. If there is an error, you get an in-place indicator or tooltip about the problem and you can fix it without the need to restart your program or loose state. 
+### Stay focused

```

Comment:
Github shows this diff in a strange way. I just wanted to suggest a different title instead of `Save time`.
I do think that vvvv might make things faster and save time - but maybe the reason is not that the compile time itself makes it longer - I rather think that the feedback cycle constantly distracts you and you lose focus? But not sure about this one to be honest.

Reply:
`Less distractions` wäre vielleicht auch ein guter Titel?